serenity.properties

```diff
@@ -1,5 +1,12 @@
-# Serenity Report properties
-# These properties are used by the Gradle build
-# serenity.project.name = Serenity Test Suite Using JUnit
-# serenity.test.root=starter
-
+serenity.project.name = Serenity_Framework
```

Comment:
serenity.properties duplicates serenity.conf settings. Please use only one e.g. serenity.conf (it's completely enough for your purpose). And let's discuss some cases when you really need both